src/renderer/components/tooltip/tooltip.tsx

```diff
@@ -54,13 +54,18 @@ export class Tooltip extends React.Component<TooltipProps> {
 
   @observable.ref elem: HTMLElement;
   @observable activePosition: TooltipPosition;
+  @observable isDomNodeRendered = false;
```

Comment:
Why do we really need this extra field? 
Can we live only `this.isVisible`? (since you have extra `get visible()` in readonly)

Reply:
This is the way to preserve tooltip animation.

1. We show DOM node first which is invisible by styles.
2. We add `visible` class on it which starts the animation.

That's why we using 2 fields in here.

Is there a way to start the css-only animation right from the element appearing?